client/my-sites/backup/clone-flow/index.tsx

```diff
@@ -0,0 +1,448 @@
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent, useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import ActivityCardList from 'calypso/components/activity-card-list';
+import AdvancedCredentials from 'calypso/components/advanced-credentials';
+import DocumentHead from 'calypso/components/data/document-head';
+import QueryRewindBackups from 'calypso/components/data/query-rewind-backups';
+import QueryRewindRestoreStatus from 'calypso/components/data/query-rewind-restore-status';
+import QueryRewindState from 'calypso/components/data/query-rewind-state';
+import BackupSuccessful from 'calypso/components/jetpack/daily-backup-status/status-card/backup-successful';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import Main from 'calypso/components/main';
+import SidebarNavigation from 'calypso/components/sidebar-navigation';
+import useRewindableActivityLogQuery from 'calypso/data/activity-log/use-rewindable-activity-log-query';
+import accept from 'calypso/lib/accept';
+import { Interval, EVERY_FIVE_SECONDS } from 'calypso/lib/interval';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+import { applySiteOffset } from 'calypso/lib/site/timezone';
+import { rewindClone } from 'calypso/state/activity-log/actions';
+import { setValidFrom } from 'calypso/state/jetpack-review-prompt/actions';
+import { requestRewindBackups } from 'calypso/state/rewind/backups/actions';
+import { getInProgressBackupForSite } from 'calypso/state/rewind/selectors';
+import getInProgressRewindStatus from 'calypso/state/selectors/get-in-progress-rewind-status';
+import getJetpackCredentials from 'calypso/state/selectors/get-jetpack-credentials';
+import getPreviousRoute from 'calypso/state/selectors/get-previous-route';
+import getRestoreProgress from 'calypso/state/selectors/get-restore-progress';
+import getRewindState from 'calypso/state/selectors/get-rewind-state';
+import getSiteGmtOffset from 'calypso/state/selectors/get-site-gmt-offset';
+import getSiteTimezoneValue from 'calypso/state/selectors/get-site-timezone-value';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import getSiteUrl from 'calypso/state/sites/selectors/get-site-url';
+import Error from '../rewind-flow/error';
+import Loading from '../rewind-flow/loading';
+import ProgressBar from '../rewind-flow/progress-bar';
+import RewindConfigEditor from '../rewind-flow/rewind-config-editor';
+import RewindFlowNotice, { RewindFlowNoticeLevel } from '../rewind-flow/rewind-flow-notice';
+import { defaultRewindConfig, RewindConfig } from '../rewind-flow/types';
+import CloneFlowStepProgress from './step-progress';
+import type { RestoreProgress } from 'calypso/state/data-layer/wpcom/activity-log/rewind/restore-status/type';
+import type { RewindState } from 'calypso/state/data-layer/wpcom/sites/rewind/type';
+import './style.scss';
+
+interface Props {
+	siteId: number;
```

Comment:
Should `siteId` be a required prop? It appears that the majority of selectors and actions depend on it.